test/new-e2e/tests/windows/temp.go

```diff
@@ -0,0 +1,23 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023-present Datadog, Inc.
+
+package windows
+
+import (
+	"strings"
+
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/utils/e2e/client"
+)
+
+// GetTemporaryFile returns a new temporary file path
+// https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/new-temporaryfile?view=powershell-7.4
+func GetTemporaryFile(vm client.VM) (string, error) {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
This seems to be used only internally by the package, you could avoid exporting it
```suggestion
// getTemporaryFile returns a new temporary file path
// https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/new-temporaryfile?view=powershell-7.4
func getTemporaryFile(vm client.VM) (string, error) {
```

Reply:
it'll be used in upcoming tests